core/operation/src/main/java/uk/gov/gchq/gaffer/operation/impl/GetWalks.java

```diff
@@ -47,10 +49,9 @@
  */
 public class GetWalks implements
         InputOutput<Iterable<? extends EntityId>, Iterable<Walk>>,
-        MultiInput<EntityId>,
-        Operations<GetElements> {
+        MultiInput<EntityId> {
 
-    private List<GetElements> operations;
+    private List<WalkDefinition> walkDefinitions;
```

Comment:
We need to come up with a way to get this to work with GraphHooks and the Graph. The Graph needs to be able to update the views on all nested operations. GraphHooks also need to be able to iterate around all operations and update them if required.